articles/Document Information Extraction with Large Language Models/README.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Document Information Extraction with Large Language Models (framework, methods,  ensemble, and evaluation)"
+author: ["Carvin Li / ML Engineer", "Anjing Wang / NLP, ML Engineer"]
+createTime: 2024-05-21
+tags: ["Information Extraction", "Large Language Model"]
+thumb: "thumb_h.png"
+thumb_b: "thumb_h.png"
```

Comment:
The `thumb_b` and `thumb` should have the different images which from the designer.  

One is a horizontal picture, one is a vertical picture

Reply:
Does the horizontal image needs to keep the words direction on the image? Or simply rotate is ok?

Reply:
Done.